backend/code-style.md

```
@@ -0,0 +1,175 @@
+
+assertSoftly 로 감싸기
+
+<hr>
+<h1><strong>3. 네이밍 규칙</strong></h1>
+
+ 구분                      | 규칙                                                       
+-------------------------|----------------------------------------------------------
+ 메서드                     | 동사 시작, 엔티티명 미사용 ex) memberService.findById(Long id)      
+ 파라미터                    | Path Variable 명확화 ex) /api/{documentId}                  
+ DTO                     | Request/Response 접미사 ⭕ 등록·수정용은 Register / Update 등 사용 가능 
+ ex) PostRegisterRequest |
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)